src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -270,8 +275,12 @@ def __await__(self) -> ResponseType:
             if self._cython_call.is_locally_cancelled():
                 raise asyncio.CancelledError()
             else:
+                call_status = self._cython_call._status
```

Comment:
Out of curiosity is `cygrpc.EOF is None`? Otherwise I do not understand how it is working considering this [1].

Would it make sense for making this piece of code change [1] for returning a EOF?

/cc @lidizheng 


[1] https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi#L330